index.html

```diff
@@ -13615,59 +13615,61 @@ <h2>ARIA Attribute Correspondence</h2>
-			<tr><td><dfn>ariaBusy</dfn></td><td><sref>aria-busy</sref></td></tr>
-			<tr><td><dfn>ariaChecked</dfn></td><td><sref>aria-checked</sref></td></tr>
-			<tr><td><dfn>ariaColCount</dfn></td><td><pref>aria-colcount</pref></td></tr>
-			<tr><td><dfn>ariaColIndex</dfn></td><td><pref>aria-colindex</pref></td></tr>
-			<tr><td><dfn>ariaColIndexText</dfn></td><td><pref>aria-colindextext</pref></td></tr>
-			<tr><td><dfn>ariaColSpan</dfn></td><td><pref>aria-colspan</pref></td></tr>
-			<tr><td><dfn>ariaControlsElements</dfn></td><td><pref>aria-controls</pref></td></tr>
-			<tr><td><dfn>ariaCurrent</dfn></td><td><sref>aria-current</sref></td></tr>
-			<tr><td><dfn>ariaDescribedByElements</dfn></td><td><pref>aria-describedby</pref></td></tr>
-			<tr><td><dfn>ariaDescription</dfn></td><td><pref>aria-description</pref></td></tr>
-			<tr><td><dfn>ariaDetailsElements</dfn></td><td><pref>aria-details</pref></td></tr>
-			<tr><td><dfn>ariaDisabled</dfn></td><td><sref>aria-disabled</sref></td></tr>
-			<tr><td><dfn>ariaErrorMessageElements</dfn></td><td><pref>aria-errormessage</pref></td></tr>
-			<tr><td><dfn>ariaExpanded</dfn></td><td><sref>aria-expanded</sref></td></tr>
-			<tr><td><dfn>ariaFlowToElements</dfn></td><td><pref>aria-flowto</pref></td></tr>
-			<tr><td><dfn>ariaHasPopup</dfn></td><td><pref>aria-haspopup</pref></td></tr>
-			<tr><td><dfn>ariaHidden</dfn></td><td><sref>aria-hidden</sref></td></tr>
-			<tr><td><dfn>ariaInvalid</dfn></td><td><sref>aria-invalid</sref></td></tr>
-			<tr><td><dfn>ariaKeyShortcuts</dfn></td><td><pref>aria-keyshortcuts</pref></td></tr>
-			<tr><td><dfn>ariaLabel</dfn></td><td><pref>aria-label</pref></td></tr>
-			<tr><td><dfn>ariaLabelledByElements</dfn></td><td><pref>aria-labelledby</pref></td></tr>
-			<tr><td><dfn>ariaLevel</dfn></td><td><pref>aria-level</pref></td></tr>
-			<tr><td><dfn>ariaLive</dfn></td><td><pref>aria-live</pref></td></tr>
-			<tr><td><dfn>ariaModal</dfn></td><td><pref>aria-modal</pref></td></tr>
-			<tr><td><dfn>ariaMultiLine</dfn></td><td><pref>aria-multiline</pref></td></tr>
-			<tr><td><dfn>ariaMultiSelectable</dfn></td><td><pref>aria-multiselectable</pref></td></tr>
-			<tr><td><dfn>ariaOrientation</dfn></td><td><pref>aria-orientation</pref></td></tr>
-			<tr><td><dfn>ariaOwnsElements</dfn></td><td><pref>aria-owns</pref></td></tr>
-			<tr><td><dfn>ariaPlaceholder</dfn></td><td><pref>aria-placeholder</pref></td></tr>
-			<tr><td><dfn>ariaPosInSet</dfn></td><td><pref>aria-posinset</pref></td></tr>
-			<tr><td><dfn>ariaPressed</dfn></td><td><sref>aria-pressed</sref></td></tr>
-			<tr><td><dfn>ariaReadOnly</dfn></td><td><pref>aria-readonly</pref></td></tr>
+			<tr><th>IDL Attribute</th><th>Reflected ARIA Content Attribute</th><th>Value type</th></tr>
+			<tr><td><dfn data-dfn-for="ARIAMixin">role</dfn></td><td><a href="#introroles">role</a></td><td><a href="#valuetype_token_list">token list</a></td></tr>
```

Comment:
`role` was actually quite difficult to look up because we don't have a table describing the value type of role, and also the reference was missing (so I've added it here).